### ISSUE TITLE
CORS Support

### ISSUE THREAD

**Issue by tgecho** [bazqux/bazqux-api#6]
Hey! I'm messing around with a little web client idea and I realized that the API currently doesn't return the `Access-Control-Allow-Origin` header required to allow javascript direct access. Is there a chance you would consider adding this header?

Thanks!

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

**Comment by vshabanov** [comment]
I don't want to allow any third party site to read user's data. Perhaps it's better to create browser extension with specific access rights to bazqux.com to implement what you want?

**Comment by vshabanov** [comment]
Maybe AWS Lambda could do the trick?

What is your little web client idea? Perhaps I can implement it.

**Comment by tgecho** [comment]
I have thought about trying out an API Gateway / Lambda combo. I won't rule it out!

To be clear, this little project is half about exercising some new tools and techniques (newer React stuff and home screen installable PWA). The rest is personal itch scratching. Your UI is perfect on the desktop, but when I skim articles on my phone, I really like an "card" interface where each card contains a couple of article summaries. I find the flicking motion to be a really efficient way to move through a lot of items without requiring precise button tapping. It's petty, but it's also an excuse to [experiment](https://css-tricks.com/practical-css-scroll-snapping/).

Feedly's app is the only example of this interaction in a feed reader I've seen in the wild.

![feedlyscroll](https://user-images.githubusercontent.com/49615/48383829-6bfcd180-e6b5-11e8-8685-44c3547ac3fb.gif)

Unfortunately I don't really like any of their actual card layout options. Also I prefer your app overall for other reasons :)

So basically I intend to make a tiny little PWA tuned to my exact picky preferences and see how fast/smooth I can make it as a pure web app. Nothing sacred. I'd be ecstatic if you saw anything of value in all this. I'll be sure to share anything I come up with!

**Comment by vshabanov** [comment]
Perhaps it's possible to do it via CSS. BazQux already have magazine view mode, which you could style to your liking. And BazQux has internal pagination (used to unload posts when you scroll too far). For magazine view there are 5 posts per page.

So you could try 
```css
body {
  scroll-snap-type: y mandatory;
}

.pagerPage {
  scroll-snap-align: start;
}
```
Perhaps it will work for you.

Interesting that one of the most popular feature requests for Feedly was continuous scrolling and they introduced it in latest versions of their app.
